src/dashboard/trulens/dashboard/pages/Evaluations.py

```diff
@@ -389,12 +403,45 @@ def highlight(s):
                             df.meta.apply(lambda m: pd.Series(m))
                         ).drop(columns="meta")
 
-                        st.dataframe(
-                            df.style.apply(highlight, axis=1).format(
-                                "{:.2f}", subset=["result"]
+                        if "groundedness" in feedback_name.lower():
+                            # Split the reasons value into separate rows and columns
+                            reasons = df["reasons"].iloc[0]
+                            statements = reasons.split("STATEMENT ")
+                            data = []
+                            for statement in statements[1:]:
+                                parts = statement.split("\n")
+                                criteria = parts[1].split(": ", 1)[1]
+                                supporting_evidence = parts[2].split(": ", 1)[1]
+                                score = int(parts[3].split(": ", 1)[1])
+                                data.append({
+                                    "Statement": criteria,
```

Comment:
This would be easier to understand if there is a sample redacted example above as a comment.

Reply:
what would the example be?

Reply:
This example is similar to the one you added in the other Pr.

"Hi. I'm here to help. The university of Washington is a public research university. UW's connections to major corporations in Seattle contribute to its reputation as a hub for innovation and technology"

- but in this case, what are the statements, critera/split

Reply:
Added comments to explain. let me know if this still needs more explanation